src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import { Component } from 'react';
+import {
+  CALCULATOROPERATORS,
+  NUM_1,
+  NUM_2,
+  NUM_3,
+  NUM_4,
+} from '../utilities/consts';
+import Button from '../button/Button';
+import './Calculator.css';
+
+export default class Calculator extends Component {
```

Comment:
- [ ] This component should be a functional component, right now it is a class component. Kindly fix that.

Reply:
Calculator component changed to a functional component👍